lib/fluent/plugin/out_forward.rb

```diff
@@ -214,22 +214,23 @@ def write(chunk)
       select_a_healthy_node{|node| node.send_data(tag, chunk) }
     end
 
-    ACKWaitingSockInfo = Struct.new(:sock, :chunk_id, :node, :time, :timeout) do
+    ACKWaitingSockInfo = Struct.new(:sock, :chunk_id, :chunk_id_base64, :node, :time, :timeout) do
```

Comment:
It's not bad idea, but it looks better for me to add these methods to `chunk`:
* `chunk_id_hex`
* `chunk_id_base64`

These values are called many times (especially _hex for logging), and should be cached not to calculate 2 or more times.
How do you think about it?